platform/openide.util.ui/test/unit/src/org/openide/xml/XMLUtilTest2.java

```diff
@@ -49,9 +49,9 @@
 import org.xml.sax.SAXParseException;
 import org.xml.sax.helpers.DefaultHandler;
 
-public class XMLUtilTest extends NbTestCase {
+public class XMLUtilTest2 extends NbTestCase {
```

Comment:
adding the `2` does disable the test?

Reply:
The 2 was intended to give the class a different name. I overlooked, that the class is not executed anymore. Will be fixed.